ocf_datapipes/training/windnet.py

```diff
@@ -142,7 +147,9 @@ def __iter__(self):
                 datasets = uncombine_from_single_dataset(dataset)
                 # print(datasets)
                 if "ecmwf" in datasets["nwp"].keys():
-                    datasets["nwp"]["ecmwf"] = potentially_coarsen(datasets["nwp"]["ecmwf"])
+                    datasets["nwp"]["ecmwf"] = potentially_coarsen(
+                        xr_data=datasets["nwp"]["ecmwf"], coarsen_to_deg=self.coarsen_to_deg
+                    )
                 # Yield a dictionary of the data, using the keys in self.keys
                 # print(datasets)
```

Comment:
i'm nitpicking, but if i'm reading this correctly there are several debugging prints left behind in this file, maybe remove them while we're here?

Reply:
yea, i can get rid of some of those prints, as we are here